components/d2l-filter-dropdown/d2l-filter-dropdown-category.js

```diff
@@ -144,6 +144,10 @@ class FilterDropdownCategory extends LocalizeStaticMixin(TabPanelMixin(LitElemen
 		});
 	}
 
+	_onSlotChange() {
+		this.dispatchEvent(new CustomEvent('d2l-filter-dropdown-category-slotchange', { bubbles: true, composed: true, detail: { categoryKey: this.key } }));
```

Comment:
I think you should be able to set this to `composed: false`, since you add the listener directly onto the `d2l-filter-dropdown` and `d2l-filter-dropdown-category` should always be in the same shadowRoot.  Nothing above that should need to know about a slot change.  A lot of the other events here also probably don't need it, but we're trying to be more aware of it going forward